pinot-controller/src/main/java/com/linkedin/pinot/controller/api/pojos/Instance.java

```diff
@@ -15,15 +15,14 @@
  */
 package com.linkedin.pinot.controller.api.pojos;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
Not sure if it's safe to change it here. Will address all fasterxml/codehaus problems in a separate PR.